app/celery/tasks.py

```diff
@@ -34,24 +33,14 @@
     put_batch_saving_bulk_created,
     put_batch_saving_bulk_processed,
 )
-from app.celery import (  # noqa: F401
```

Comment:
Any idea why these were imported if not used?

Reply:
unsure :/